pages/case_studies/case_studies.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Case Studies
+permalink: /case-studies
+---
```

Comment:
Assuming this and GWF page should not published? Assuming content will come from gwf-case-study branch.

Reply:
Not yet no, we'll merge all that in when it's ready.

I also don't think we need sub-navigation for case studies. Just a link to a case studies page which can list the case studies with a bit of blurb about it, linking to each case study from there.

Reply:
Have set case studies pages to be unpublished and removed from navigation. Also removed the sub nav for case studies